pkg/operator/ceph/csi/spec.go

```diff
@@ -676,12 +676,29 @@ func (r *ReconcileCSI) startDrivers(ver *version.Info, ownerInfo *k8sutil.OwnerI
 	return nil
 }
 
-func (r *ReconcileCSI) stopDrivers(ver *version.Info) error {
+// return false is rook operator is not the owner of csi driver,
+// so csi driver do not get deleted as it isn't deployed by
+// rook operator and someone externally installed it
+func (r *ReconcileCSI) validateDriverOwner(ownerInfo *k8sutil.OwnerInfo, deploymentName string) bool {
```

Comment:
`validateDriverOwner` is not accurate you might need to use `OwnedByRook` or something else?

Reply:
updated to `driverOwnedByRook`, thanks for the suggestion